icons/example.jsx

```diff
@@ -87,7 +89,7 @@ export default {
     )
   }, {
     title: 'Small',
-    require: icons.tiny.map(({ name }) => `import ${name} from '@klarna/ui/icons/${name}'`).join('\n'),
+    require: icons.tiny.map(({ displayName }) => `import ${displayName} from '@klarna/ui/icons/${displayName}'`).join('\n'),
```

Comment:
I'm seeing the output of this as `import undefined from '@klarna/ui/icons/undefined` for every icon. I think since we are not setting it manually, it's just `name`.